pkgs/by-name/sp/speakersafetyd/package.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, lib
+, rustPlatform
+, fetchCrate
+, pkg-config
+, alsa-lib
+, rust
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "speakersafetyd";
+  # tracking: https://src.fedoraproject.org/rpms/rust-speakersafetyd
+  version = "0.1.9";
+
+  src = fetchCrate {
+    inherit pname version;
+    hash = "sha256-I1fL1U4vqKxPS1t6vujMTdi/JAAOCcPkvUqv6FqkId4=";
+  };
+  cargoHash = "sha256-Adwct+qFhUsOIao8XqNK2zcn13DBlQNA+X4aRFeIAXM=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ alsa-lib ];
+
+  postPatch = ''
+    substituteInPlace speakersafetyd.service --replace "/usr" "$out"
+    substituteInPlace Makefile --replace "target/release" "target/${rust.lib.toRustTargetSpec stdenv.hostPlatform}/$cargoBuildType"
+  '';
+
+  installFlags = [
+    "DESTDIR=${placeholder "out"}"
```

Comment:
This'll work fine for this Makefile, but usually we want to avoid DESTDIR, because it generally means "install files to this directory, but if you compile in paths into your program, don't include the destdir prefix".

Would you mind doing e.g. `BINDIR=${placeholder "out"}/bin`, just so people reading this don't get used to the idea that using DESTDIR is the right thing to do in Nixpkgs in general?

BTW: environment variables are interpreted in Make flags, so it's possible to do e.g. `$(out)` instead of `${placeholder "out"}`, and that means things won't get weird if somebody tries to debug the package in a nix-shell, where placeholders aren't processed.

Reply:
I was hoping to avoid setting all the *DIR parameters, but that seems like a good reason to not use DESTDIR.

Done